src/announcement.iced

```diff
@@ -12,7 +12,9 @@ exports.Announcement = class Announcement extends Base
 
   _type : () -> constants.sig_types.announcement
 
-  _json : () -> 
+  _required_stanzas : () -> super.concat(["announcement"])
```

Comment:
Ah, yea, I think it's a "try to be more like Ruby" thing, `super` implicitly adds all the arguments. Agreed that explicit is better though, should be updated!